### DISCUSSION TITLE
Math Component6.2. Updating the Topology

### DISCUSSION THREAD

**Discussion by imlixixi** [nasa/fprime#1318] [Q&A]
I am trying to complete the fpp-check as directed in the Math Component Tutorial but I am getting an error when running the command.
fprime-util fpp-check -u unconnected.txt
[INFO] Updating fpp locations file and build cache. This may take some time.
[ERROR] No fpp dependency memo found. Perhaps directory is outside build?
(I have added the fpp-tools's to the env)
<img width="1173" alt="截屏2022-03-10 下午4 45 05" src="https://user-images.githubusercontent.com/51080083/157624340-cbd5de49-1dee-4559-9120-d4b4888ed921.png">

**Comment by Emil808** [comment]
Try these in /Ref/ before trying `fprime-util fpp-check`
```
fprime-util generate
fprime-util build
```

**Comment by ThibFrgsGmz** [comment]
Duplicate thread: https://github.com/nasa/fprime/discussions/1307
